test/verk_test.exs

```diff
@@ -129,15 +130,15 @@ defmodule VerkTest do
   end
 
   test "schedule a job with a jid, a queue and a perform_in passing a redis connection" do
-    now = Timex.DateTime.now
-    perform_at = Timex.DateTime.shift(now, days: 1)
+    now = Time.now
+    perform_at = Time.shift(now, 100, :seconds)
```

Comment:
I don't get why 1 day became 100 seconds here? Any reason why this needs to be changed?